### PR TITLE
Turn into a gradle project for local development

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: google-form-post.iml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      <excludeFolder url="file://$MODULE_DIR$/build/test-results" />
+      <excludeFolder url="file://$MODULE_DIR$/build/tmp" />
+    </content>
+    <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Gradle: junit:junit:4.12@jar" level="project" />
+    <orderEntry type="library" name="Gradle: org.apache.cordova:framework-7.1.0" level="project" />
+    <orderEntry type="library" name="Gradle: org.hamcrest:hamcrest-core:1.3@jar" level="project" />
+  </component>
+</module>
```
Comment (MartinEden) [inline google-form-post.iml]: We've tended not to check in the IntelliJ project files. But we can do so here, if you think it's a good idea.

**Review by MartinEden — CHANGES_REQUESTED** [review]
Now that it's gradle, can't you use Gradle to manage the JUnit dependency, instead of checking in a JAR?

**Comment by MartinEden** [comment]
I'm currently installing Android Studio. I'll give more feedback once I've got that working.

**Comment by hillalex** [comment]
Re JUnit JAR, yes, this PR is removing that file!

File: google-form-post.iml (hunk 1 above)
Comment (hillalex) [reply google-form-post.iml]: No particular reason, I'll remove

**Comment by MartinEden** [comment]
Ah, okay. I wasn't sure.

This looks good. One small request: Could you add to the README what people need to do to get this compiling? For me it was:

1. Install and run Android Studio (making sure an SDK is installed)
2. Add `export ANDROID_HOME=/some/path` to my `.profile` file
3. Run `./gradlew build`

**Review by MartinEden — APPROVED** [review]

**Pull request merged**